BlackNectar/BlackNectar/CustomView.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  CustomerView.swift
+//  BlackNectar
+//
+//  Created by Wellington Moreno on 3/2/17.
+//  Copyright © 2017 BlackSource. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+@IBDesignable class CustomView : UIView {
+    
+    override init(frame: CGRect) {
+        
+        super.init(frame: frame)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        
+        super.init(coder: aDecoder)
+        
+    }
+    
+    override func prepareForInterfaceBuilder() {
+        
+        updateView()
+        
+    }
+    
+    @IBInspectable var circular: Bool = false {
+        
+        didSet {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var borderWidth: CGFloat = 1 {
+        
+        didSet {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var borderColor: UIColor = UIColor.white {
+        
+        didSet {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var shouldRasterize: Bool = false {
+        
+        didSet {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var shadowOffSet: CGSize = CGSize(width: 3, height: 0) {
+        
+        didSet {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var cornerRadius: CGFloat = 5 {
+        
+        didSet {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var rotation: CGFloat = 0 {
+        
+        didSet {
+            updateView()
+        }
+    }
+    
+    func updateView() {
+        
+        layer.cornerRadius = cornerRadius
+        layer.borderWidth = borderWidth
+        layer.borderColor = borderColor.cgColor
+        layer.masksToBounds = true
+        
+        if shouldRasterize {
+            layer.rasterizationScale = UIScreen.main.scale
+        }
+        
+        var rotationDegrees = Int(rotation)
```

Comment:
You can probably just condense this to

```swift
rotationDegrees = rotation % 360
```

That should actually take care of all cases.